src/app/controllers/CartController.ts

```diff
@@ -288,5 +293,141 @@ export class CartController {
     });
   }
 
+  /**
+   * Get Cart order total for a customer
+   * @param {string} cartId
+   * @param {string} sharedToken
+   * @returns {Promise<number>}
+   */
+  getCartOrderTotal(cartId: string, sharedToken?: string): Promise<number> {
```

Comment:
![image](https://user-images.githubusercontent.com/7469722/104944332-e7303100-596b-11eb-8272-b6a72408e29c.png)

Why do we need this when we have the Cart Total field btw?